code/game/objects/items/stacks/sheets/mineral.dm

```diff
@@ -490,3 +490,29 @@ GLOBAL_LIST_INIT(metalhydrogen_recipes, list(
 /obj/item/stack/sheet/mineral/metal_hydrogen/get_main_recipes()
 	. = ..()
 	. += GLOB.metalhydrogen_recipes
+
+GLOBAL_LIST_INIT(zaukerite_recipes, list(
+	new /datum/stack_recipe("floor tile", /obj/item/stack/tile/material, 1, 4, 20),
+	new /datum/stack_recipe("chair", /obj/structure/chair/greyscale, 1, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("table", /obj/structure/table/greyscale, 1, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("toilet", /obj/structure/toilet/greyscale, 1, one_per_turf = TRUE, on_floor = TRUE),
+	new /datum/stack_recipe("carving block", /obj/structure/carving_block, req_amount = 15, res_amount = 1),
```

Comment:
why tf copypaste these

Reply:
fun i suppose

Reply:
fix this dumbass

Reply:
what there is to fix?

Reply:
this isnt how those are supposed to be applied

Reply:
i just followed the other minerals

Reply:
should be fixed, thanks for the patience!